react/features/base/jwt/components/CallOverlay.js

```diff
@@ -77,6 +77,18 @@ class CallOverlay extends Component<*, *> {
         _callee: PropTypes.object
     };
 
+    /**
+     * Check if this overlay needs to be rendered. This function will be called
+     * by the {@code OverlayContainer}.
+     *
+     * @param {Object} state - The redux state.
+     * @returns {boolean} - True if this overlay needs to be displayed, false
+     * otherwise.
+     */
+    static needsDisplay(state) {
```

Comment:
The method's jsdoc says "needs to be rendered" and the method name is `needsDisplay`. The `@returns` says "needs to be displayed." In accord with our coding style and for the sake of simplicity, could we please stick to one name. I guess "render" is more appropriate than "display" given that we're dealing with React.

Reply:
Agreed!